### PR TITLE
Clarify that TraceContext/Baggage propagators are required.

### LINKED ISSUE #1093: There is only a MAY requirement for W3C propagator
The spec says in https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#included-propagators that a W3C propagator MAY be included in the API. It says in https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context/api-propagators.md#propagators-distribution that Jaeger and B3 MUST be available and maintained by OpenTelemetry.

It seems that the W3C propagator also needs to be included into the ~~first~~ second list, to make it a MUST for language SIGs.

(from https://github.com/open-telemetry/opentelemetry-specification/issues/1070#issuecomment-708572787)

CC @carlosalberto @SergeyKanzhelev

### PR DESCRIPTION
**Pull request by carlosalberto**
Fixes #1093 

cc @Oberon00

### CODE REVIEW

File: specification/context/api-propagators.md [hunk 1]
```diff
@@ -312,8 +312,12 @@ Required parameters:
 The official list of propagators that MUST be maintained by the OpenTelemetry
 organization and MUST be distributed as OpenTelemetry extension packages:
 
-* [B3](https://github.com/openzipkin/b3-propagation)
-* [Jaeger](https://www.jaegertracing.io/docs/latest/client-libraries/#propagation-format)
+* [W3C TraceContext](https://www.w3.org/TR/trace-context/), MAY alternatively
+  be distributed as part of the OpenTelemetry API.
+* [W3c Baggage](https://w3c.github.io/baggage), MAY alternatively
+  be distributed as part of the OpenTelemetry API.
+* [B3](https://github.com/openzipkin/b3-propagation).
```
Comment (Oberon00) [inline specification/context/api-propagators.md:319]:
```suggestion
* [B3](https://github.com/openzipkin/b3-propagation)
```

This makes the diff a bit confusing

File: specification/context/api-propagators.md [hunk 2]
```diff
@@ -312,8 +312,12 @@ Required parameters:
 The official list of propagators that MUST be maintained by the OpenTelemetry
 organization and MUST be distributed as OpenTelemetry extension packages:
 
-* [B3](https://github.com/openzipkin/b3-propagation)
-* [Jaeger](https://www.jaegertracing.io/docs/latest/client-libraries/#propagation-format)
+* [W3C TraceContext](https://www.w3.org/TR/trace-context/), MAY alternatively
```
Comment (Oberon00) [inline specification/context/api-propagators.md:315]:
```suggestion
* [W3C TraceContext](https://www.w3.org/TR/trace-context/). MAY alternatively
```

File: specification/context/api-propagators.md [hunk 3]
```diff
@@ -312,8 +312,12 @@ Required parameters:
...
+  be distributed as part of the OpenTelemetry API.
+* [W3c Baggage](https://w3c.github.io/baggage), MAY alternatively
```
Comment (Oberon00) [inline specification/context/api-propagators.md:317]:
```suggestion
* [W3c Baggage](https://w3c.github.io/baggage). MAY alternatively
```

**Review by Oberon00 — APPROVED** [review]

**Review by jkwatson — APPROVED** [review]

File: specification/context/api-propagators.md [hunk 4]
```diff
@@ -312,8 +312,12 @@ Required parameters:
 The official list of propagators that MUST be maintained by the OpenTelemetry
 organization and MUST be distributed as OpenTelemetry extension packages:
 
+* [W3C TraceContext](https://www.w3.org/TR/trace-context/). MAY alternatively
+  be distributed as part of the OpenTelemetry API.
+* [W3c Baggage](https://w3c.github.io/baggage). MAY alternatively
+  be distributed as part of the OpenTelemetry API.
 * [B3](https://github.com/openzipkin/b3-propagation)
```
Comment (yurishkuro) [inline specification/context/api-propagators.md:319]:
```suggestion
* [B3](https://github.com/openzipkin/b3-propagation).
```

File: specification/context/api-propagators.md [hunk 5]
```diff
@@ -312,8 +312,12 @@ Required parameters:
 The official list of propagators that MUST be maintained by the OpenTelemetry
 organization and MUST be distributed as OpenTelemetry extension packages:
 
+* [W3C TraceContext](https://www.w3.org/TR/trace-context/). MAY alternatively
+  be distributed as part of the OpenTelemetry API.
+* [W3c Baggage](https://w3c.github.io/baggage). MAY alternatively
```
Comment (yurishkuro) [inline specification/context/api-propagators.md:317]:
```suggestion
* [W3C Baggage](https://w3c.github.io/baggage). MAY alternatively
```

**Review by yurishkuro — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

File: specification/baggage/api.md [hunk 6]
```diff
@@ -120,6 +120,8 @@ The API layer MAY include the following `Propagator`s:
 
 * A `TextMapPropagator` implementing the [W3C Baggage Specification](https://w3c.github.io/baggage).
 
+See [propagators distribution](../context/api-propagators.md#propagators-distribution).
```
Comment (arminru) [inline specification/baggage/api.md:123]:
```suggestion
See [Propagators Distribution](../context/api-propagators.md#propagators-distribution)
for how propagators are to be distributed.
```

File: specification/trace/api.md [hunk 7]
```diff
@@ -735,6 +735,8 @@ The API layer MAY include the following `Propagator`s:
 
 * A `TextMapPropagator` implementing the [W3C TraceContext Specification](https://www.w3.org/TR/trace-context/).
 
+See [propagators distribution](../context/api-propagators.md#propagators-distribution).
```
Comment (arminru) [inline specification/trace/api.md:738]:
```suggestion
See [Propagators Distribution](../context/api-propagators.md#propagators-distribution)
for how propagators are to be distributed.
```

File: specification/baggage/api.md [hunk 8]
```diff
@@ -120,6 +120,8 @@ The API layer MAY include the following `Propagator`s:
```
Comment (arminru) [inline specification/baggage/api.md:120]: It still says MAY above. We could say "The API layer or an extension package MUST ..."

File: specification/trace/api.md [hunk 9]
```diff
@@ -735,6 +735,8 @@ The API layer MAY include the following `Propagator`s:
```
Comment (arminru) [inline specification/trace/api.md:735]: It still says MAY above. We could say "The API layer or an extension package MUST ..."

**Review by arminru — APPROVED** [review]

File: specification/trace/api.md (hunk 9 above)
Comment (carlosalberto) [reply specification/trace/api.md:735]: Actually that's not the requirement ;) Currently we ask for a MUST when it comes to offering the propagator, but the agreement (from the MAY) is that it _may be located in the API itself_.

Hope that makes sense ;)

Comment (arminru) [reply specification/trace/api.md:735]: The wording I proposed is based on the requirements we define in the `api-propagators.md`:

> MUST be distributed as OpenTelemetry extension packages:

and

> MAY alternatively be distributed as part of the OpenTelemetry API.

The combination of "The API layer MAY include" and the statements in `api-propagators.md` yields the same requirement (i.e., a MUST in either of the places), so we can also keep it as-is, but the wording I suggested would make it more obvious IMHO.

Comment (carlosalberto) [reply specification/trace/api.md:735]: Oh, I got it. Yes, it makes it double explicit. Will add ;)

Comment (carlosalberto) [reply specification/trace/api.md:735]: Done ;)

**Comment by carlosalberto** [comment]
Merging as this is merely an editorial change with a clarification.

**Pull request merged**